packages/dialog/src/__private/dialog-toolbar.tsx

```diff
@@ -29,17 +31,26 @@ const Div = styled.div<UiReactPrivateElementProps>`
   ${(props) => getThemeStyling(props.customTheme, props.selectedTheme, dialogToolbarMapper)}
 
   border-style: solid;
-  border-width: 0 0 5px 0;
+  border-width: 0 0 2px 0;
   padding-top: 5px;
   padding-bottom: 5px;
 `;
 
-export const DialogToolbar: React.FC<DialogToolbarProps> = ({ closeCB, hideCloseIcon, title }: DialogToolbarProps) => {
+export const DialogToolbar: React.FC<DialogToolbarProps> = ({
+  closeLabel,
+  closeCB,
+  hideCloseIcon,
+  title,
+}: DialogToolbarProps) => {
   const theme = React.useContext(ThemeContext);
 
   return (
     <Div customTheme={theme.theme} selectedTheme={theme.selectedTheme}>
-      {!hideCloseIcon && <Button onClick={closeCB}>❌</Button>}
+      {!hideCloseIcon && (
+        <Button onClick={closeCB} aria-label={closeLabel}>
+          <UiIcon icon="Cross" />
```

Comment:
is it worth at some point making the icon configurable ?

Reply:
That's a good idea, will add it to the backlog.

Reply:
https://github.com/inavac182/uireact/issues/156